src/docs/release/breaking-changes/flutter-lints-package.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Introducing package:flutter_lints
+description: Migrate to package:flutter_lints to get the latest set of recommended lints, which encourage good coding practices.
+---
+
+## Summary
+
+The [`package:flutter_lints`][] defines the latest set of recommended lints that
+encourage good coding practises for Flutter apps, packages, and plugins. Projects
```

Comment:
I *think* this should be "practices".

Reply:
Correct, we use American English, not British English on our developer sites:  https://developers.google.com/style/spelling?hl=en. :D